README.md

```diff
@@ -80,13 +80,19 @@ $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
 
 ### Installing mbed CLI
 
+#### Windows Installer
```

Comment:
Please lower case "installer" because we use sentence case for headings except for the very first heading (title) at the top of the page.